### DISCUSSION TITLE
work/V6.0.01.00/70115%E3%80%90%E6%B2%B3%E5%8D%97%E3%80%91%E7%BB%BC%E5%90%88%E6%90%9C%E7%B4%A2%E5%A2%9E%E5%8A%A0%E5%8C%B9%E9%85%8D%E5%BA%A6%E8%A7%84%E5%88%99%E5%B9%B6%E5%8E%BB%E9%87%8D/%E4%BC%98%E5%8C%96ES%E5%85%B3%E9%94%AE%E8%AF%8D%E6%90%9C%E7%B4%A2

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#701] [Announcements]
# work/V6.0.01.00/70115%E3%80%90%E6%B2%B3%E5%8D%97%E3%80%91%E7%BB%BC%E5%90%88%E6%90%9C%E7%B4%A2%E5%A2%9E%E5%8A%A0%E5%8C%B9%E9%85%8D%E5%BA%A6%E8%A7%84%E5%88%99%E5%B9%B6%E5%8E%BB%E9%87%8D/%E4%BC%98%E5%8C%96ES%E5%85%B3%E9%94%AE%E8%AF%8D%E6%90%9C%E7%B4%A2

1.从多个字段搜索 目前商品上架后，会根据商品名称、品目、品牌、型号、参数等信息生成一个字段 keyword 作为查询时的字段，以便覆盖全部信息。 ```json GET goodspriceindex/_search { "query": { "match": { "keyword": "便携式计算机" } } } ``` 但是不能够体现字段的优先级...

http://localhost:8080/work/V6.0.01.00/70115%E3%80%90%E6%B2%B3%E5%8D%97%E3%80%91%E7%BB%BC%E5%90%88%E6%90%9C%E7%B4%A2%E5%A2%9E%E5%8A%A0%E5%8C%B9%E9%85%8D%E5%BA%A6%E8%A7%84%E5%88%99%E5%B9%B6%E5%8E%BB%E9%87%8D/%E4%BC%98%E5%8C%96ES%E5%85%B3%E9%94%AE%E8%AF%8D%E6%90%9C%E7%B4%A2.html

**Comment by AriesYB** [comment]
要得
